bin/crew

```diff
@@ -909,6 +909,17 @@ def prepare_package(destdir)
     system 'find . -type l >> ../filelist'
     system 'cut -c2- ../filelist > filelist'
 
+    # check for FHS3 compliance
+    @fhs_compliant_prefix = ['bin', 'etc', 'include', 'lib', 'libexec', 'opt', 'sbin', 'share', 'var']
+    @fhs_compliant_prefix.append(ARCH_LIB) unless ARCH_LIB == 'lib'
+    Dir.foreach(CREW_DEST_PREFIX) do |filename|
+      next if filename == '.' or filename == '..'
+      unless @fhs_compliant_prefix.include?(filename)
+        puts "Unable to complete this build because #{CREW_PREFIX}/#{filename} in #{@pkg.name} is not FHS3 compliant.".lightred
+        abort unless ENV['CREW_FHS_INCOMPLIANCE_ONLY_ADVISORY'] == '1'
```

Comment:
`CREW_FHS_NONCOMPLIANCE_ADVISORY_ONLY`?

Reply:
Noncompliance seems more correct, you're right. Although both are words, incompliance implies cognitive thought of opposition.

Reply:
9d8e109